analysis/src/python/data_collection/extend_dataset.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import sys
+
+from analysis.src.python.data_collection.hyperskill.hyperskill_client import HyperskillClient, ObjectClass
+import pandas as pd
+
+
+def chunks(lst, n):
+    for i in range(0, len(lst), n):
+        yield lst[i:i + n]
+
+
+def extend_hyperskill_dataset(csv_file_path: str, obj_class: str):
+    api = HyperskillClient()
+    df = pd.read_csv(csv_file_path)
+
+    if obj_class == ObjectClass.STEP:
+        ids = list(set(df['step_id'].values))
+        for ids_chunk in chunks(ids, 20):
+            api.get_steps(ids=ids_chunk, save_to_csv=True)
+
+    if obj_class == ObjectClass.TRACK:
+        ids = list(set(df['id'].values))
+        for ids_chunk in chunks(ids, 20):
+            api.get_tracks(ids=ids_chunk, save_to_csv=True)
```

Comment:
Also, it seems you can create a function with two arguments - columns and a functions from API instead of code duplicates